app/views/podcast_metrics/_main_card.html.erb

```diff
@@ -2,35 +2,35 @@
   <div class="card shadow border-0">
     <div class="card-header card-header-light d-flex justify-content-between">
       <ul class="col-8 nav nav-pills nav-justified card-header-pills">
-        <li class="nav-item"><button class="nav-link active" id="downloads-tab" data-bs-toggle="tab" data-bs-target="#downloads" type="button" role="tab" aria-controls="downloads" aria-selected="true" data-action="apex-nav#changeTab" data-apex-nav-card-param="main" data-apex-nav-tab-param="downloads"><%= t(".tabs.downloads") %></button></li>
-        <li class="nav-item"><button class="nav-link" id="episodes-tab" data-bs-toggle="tab" data-bs-target="#episodes" type="button" role="tab" aria-controls="episodes" aria-selected="false" data-action="apex-nav#changeTab" data-apex-nav-tab-param="episodes"><%= t(".tabs.episodes") %></button></li>
-        <li class="nav-item"><button class="nav-link" id="uniques-tab" data-bs-toggle="tab" data-bs-target="#uniques" type="button" role="tab" aria-controls="uniques" aria-selected="false" data-action="apex-nav#changeTab" data-apex-nav-tab-param="uniques"><%= t(".tabs.uniques") %></button></li>
-        <li class="nav-item"><button class="nav-link" id="dropdays-tab" data-bs-toggle="tab" data-bs-target="#dropdays" type="button" role="tab" aria-controls="dropdays" aria-selected="false" data-action="apex-nav#changeTab" data-apex-nav-tab-param="dropdays"><%= t(".tabs.dropdays") %></button></li>
+        <li class="nav-item"><button class="nav-link" id="downloads-tab" data-bs-toggle="tab" data-bs-target="#downloads" type="button" role="tab" aria-controls="downloads" aria-selected="false" data-action="apex-nav#displayCard" data-apex-nav-card-param="main" data-apex-nav-tab-param="downloads" data-apex-nav-target="tab" data-card="main" data-tab="downloads"><%= t(".tabs.downloads") %></button></li>
```

Comment:
so here i remove `active` and change `aria-selected` to false so that there is technically no default tab to begin with, and the tab to be displayed relies on the `main_card` param.

this works well for the most part, but where i'm stuck (i think) is because of bootstrap. changing tabs with `data-bs-toggle` doesn't seem to advance browser history, so the params don't immediately change when just changing from tab to tab. one of the other form inputs needs to also change because those inputs are the ones submitting a new request each time.